api/src/main/java/io/strimzi/api/kafka/model/KafkaExporterResources.java

```diff
@@ -49,4 +49,18 @@ public static String serviceName(String kafkaClusterName) {
     public static String secretName(String clusterName) {
         return deploymentName(clusterName) + "-certs";
     }
+
+    /**
+     * Returns the name of the Kafka Exportor {@code NetworkPolicy} for a {@code Kafka} cluster of the given name.
+     * This {@code NetworkPolicy} will only exist if {@code Kafka.spec.kafka-exporter} is configured in the
+     * {@code Kafka} resource with the given name.
+     *
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     *
+     * @return The name of the corresponding Kafka Exportor {@code NetworkPolicy}.
+     */
+    public static String networkPolicyName(String clusterName) {
+        return clusterName + "-network-policy-kafka-exporter";
+    }
```

Comment:
I don't think this is really used and can be removed, or?

Reply:
removed..